categories_pl.py

```diff
@@ -52,8 +96,9 @@ def single_game(alphabet, players):
         answers = get_answers(letter, categories)
```

Comment:
Maybe it would even be good to have answers connected to players from the start? This connection is list-index based, easy to go out of sync if one user is skipped or sth like that

Reply:
Yup, that would be a smart thing to do, but I'm not so sure on how to do that :sob:

Reply:
Let's leave as is for now and write a ticket to refactor later :D